spec.html

```diff
@@ -36697,8 +37278,8 @@ <h1>RegExp.prototype [ @@replace ] ( _string_, _replaceValue_ )</h1>
           1. Let _flags_ be ? ToString(? Get(_rx_, *"flags"*)).
           1. If _flags_ contains *"g"*, let _global_ be *true*. Otherwise, let _global_ be *false*.
           1. If _global_ is *true*, then
-            1. If _flags_ contains *"u"*, let _fullUnicode_ be *true*. Otherwise, let _fullUnicode_ be *false*.
             1. Perform ? Set(_rx_, *"lastIndex"*, *+0*<sub>𝔽</sub>, *true*).
+          1. If _flags_ contains *"u"* or _flags_ contains *"v"*, let _fullUnicode_ be *true*. Otherwise, let _fullUnicode_ be *false*.
```

Comment:
Is there a reason to pull this out of the conditional? It's only referred to under the same conditions below. This seems like an unrelated editorial change.

Reply:
Addressed as part of https://github.com/mathiasbynens/ecma262/pull/21. PTAL